reloc/build_reloc.sh

```diff
@@ -37,6 +37,7 @@ print_usage() {
     echo "  --clean                 clean build directory"
     echo "  --compiler PATH         C++ compiler path"
     echo "  --c-compiler PATH       C compiler path"
+    echo "  --disable-update-version disable updating SCYLLA-VERSION-FILE and SCYLLA-RELEASE-FILE"
```

Comment:
Maybe add here reason? Why should we avoid updating them?
... in order to avoid different revision number on build packages if compiled around midnight.

Reply:
I'll update the description.